tool/README.md

```diff
@@ -62,9 +61,8 @@ Review/update `CHANGELOG.md`:
 3. Verify each item is a complete sentence, written as though it was an order, and there is no syntax errors.
 4. Create draft PR for the branch and add the item for it to the top.
 
-### Test the `$DEVTOOLS_RELEASE_BRANCH`
+### Test the release PR
 
-- Checkout the `$DEVTOOLS_RELEASE_BRANCH`,
 - Build the DevTools binary and run it from your local Dart SDK.
    - From the main devtools/ directory.
```

Comment:
One thing I just thought of: we should make sure the build_e2e.dart script points (maybe optionally?) to the dart sdk that is included with flutter. I think as is, there will be a dart version mismatch between the devtools server and the dart sdk that we use to run devtools_app 🤔

Reply:
Would it be ok for us to make a follow up issue for this one? Based on your description this would already be an issue right?

Reply:
yep 👍

Reply:
Issue created :D
https://github.com/flutter/devtools/issues/5594